src/Serilog.Sinks.Seq/Sinks/Seq/HttpLogShipper.cs

```diff
@@ -280,6 +282,58 @@ void OnTick()
             }
         }
 
+        private const string _invalidPayloadFilePrefix = "invalid-";
+        private async Task FailResponse(HttpResponseMessage result, string payload)
+        {
+            var invalidPayloadFilename = $"{_invalidPayloadFilePrefix}{result.StatusCode}-{Guid.NewGuid():n}.json";
+            var invalidPayloadFile = Path.Combine(_logFolder, invalidPayloadFilename);
+            var resultContent = await result.Content.ReadAsStringAsync();
+            SelfLog.WriteLine("HTTP shipping failed with {0}: {1}; dumping payload to {2}", result.StatusCode, resultContent, invalidPayloadFile);
+            var bytesToWrite = Encoding.UTF8.GetBytes(payload);
+            if (_errorFileSizeLimitBytes.HasValue)
+            {
+                CleanupErrorFiles(_errorFileSizeLimitBytes.Value - bytesToWrite.Length, _logFolder);
+            }
+            IOFile.WriteAllBytes(invalidPayloadFile, bytesToWrite);
+        }
+
+        private static void CleanupErrorFiles(long maxNumberOfBytesToRetain, string logFolder)
```

Comment:
`CleanUpInvalidPayloadFiles()`? Minor tweak but might help keep the term for the files consistent throughout.